readme.md

```diff
@@ -107,6 +107,29 @@ It's really important that you keep your server time in sync with some NTP serve
 
     ntpdate ntp.ubuntu.com
 
+## Validation Window
+
+To avoid problms with clocks that are slightly out of sync, we do not check against the current key only but also consider `$window` keys each from the past and future. You can pass `$window` as optional third parameter to `verifyKey`, it defaults to `4`. A new key is generated every 30 seconds, so this window includes keys from the previous two and next two minutes.

```

Comment:
Please run this text through spell checker please. At least `problms` should be written as `problems`.